cop-import-export.php

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I think we should keep it modular, so you can control what features you want in plugin, you see the custom-options-plus.php is getting bigger and hard to view, thats why we should divide it in small files, so if you change some feature in the future, you will see clearly where you changed without mix with the other parts (modularization is good). Yes, the start of this process can be confusing because you will move some code to another file but in the future would be a good harvest.